wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -86,7 +86,7 @@ <h1>
         <h2>Talk Notes</h2>
         <p>(The following is not visible to attendees.)</p>
       {% endblocktrans %}
-      <div class="well">
+      <div id="notes">
```

Comment:
Shouldn't these sections also become cards, to better how things behaved with bootstrap 3?

Reply:
Yeah, I went through that thought process too. Cards didn't look right for the abstract, in the debconf17 theme, at the time.

Reply:
But for notes, they may make sense